OWNERS

```diff
@@ -0,0 +1,3 @@
+---
+teams:
+- Compute Infra <compute-infra@yelp.com>
```

Comment:
we can probably merge this after a small change:
```suggestion
- Compute Infrastructure Platform Experience
```

(we no longer require emails in OWNERS files internally so we can omit it and potentially save ourselves some spam should we ever allow these address to receive external mail - not that these being well-formed matters since we're not indexing our OSS yet, tho :p)